tests/cupy_tests/test_ndim.py

```diff
@@ -0,0 +1,67 @@
+import numpy
+import unittest
+
+import cupy
+from cupy import testing
+
+
+@testing.gpu
+class TestNdim(unittest.TestCase):
+
+    @testing.numpy_cupy_equal()
+    def test_ndim_ndarray1d(self, xp):
+        return xp.ndim(xp.arange(5))
+
+    @testing.numpy_cupy_equal()
+    def test_ndim_ndarray2d(self, xp):
+        assert xp.ndim(xp.ones((2, 4))) == 2
```

Comment:
I think with `numpy_cupy_equal` we should return instead of asserting. I can fix this later.

Reply:
See #3890.